jerry-debugger/jerry-client-ws.py

```diff
@@ -167,10 +167,18 @@ def exec_command(self, args, command_id):
     def do_continue(self, args):
         """ Continue execution """
         self.exec_command(args, JERRY_DEBUGGER_CONTINUE)
+        print("(jerry-debugger) Use \'stop\' command to pause the debugger.")
+        self.cmdloop()
```

Comment:
Is this a recursive call? My problem is that the network message processing should not be stop when continue is executed.